src/character_body.cpp

```diff
@@ -484,7 +485,8 @@ void Character::update_bodytemp()
             continue;
         }
 
-        // This adjusts the temperature scale to match the bodytemp scale,
+        // 0.01c is 5 body temp unit so it needs to times 5 to scale to match the bodytemp scale,
+        // but this will change current behavior,
```

Comment:
this is a very weird comment, what current behaviour? If this get merged the comment would stay and the "current behaviour" would be whatever this PR implements

Reply:
I think I'll just revert these comments since the unit may not be right is already mentioned #58961 and the comment may be confuse.